src/_text.scss

```diff
@@ -0,0 +1,91 @@
+.ta-left {
+  text-align: left;
+}
+
+.ta-center {
+  text-align: center;
+}
+
+.ta-right {
+  text-align: right;
+}
+
+@each $breakpoint-name, $breakpoint-value in $breakpoints {
+  @media #{$breakpoint-value} {
+    .#{$breakpoint-name}--ta-left {
+      text-align: left;
+    }
+
+    .#{$breakpoint-name}--ta-center {
+      text-align: center;
+    }
+
+    .#{$breakpoint-name}--ta-right {
+      text-align: right;
+    }
+  }
+}
+
+.td-none {
```

Comment:
I wonder how this name will play out if we introduce classes for table data elements 🤔

Reply:
what would those classes be?